packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -56,6 +59,34 @@ class NetworkController extends DisposableController
     );
     subscribeToFilterChanges();
   }
+  List<DartIOHttpRequestData>? _httpRequests;
+
+  String? exportAsHarFile() {
+    final exportController = ExportController();
+
+    _httpRequests =
+        filteredData.value.whereType<DartIOHttpRequestData>().toList();
+
+    if (_httpRequests!.isEmpty) {
+      debugPrint('No valid request data to export');
+      return '';
+    }
+
+    try {
+      if (_httpRequests != null && _httpRequests!.isNotEmpty) {
+        // Build the HAR object
+        final har = buildHar(_httpRequests!);
+        debugPrint('data is ${json.encode(har)}');
+        return exportController.downloadFile(
```

Comment:
you can create the export controller here directly instead of creating and storing in a variable above:
`ExportController().downloadFile(...)`

Reply:
done